docs/modules/deployment/pages/core/getting-started.adoc

```diff
@@ -39,6 +43,79 @@ endif::[]
 
 include::../time-sync.adoc[]
 
+[[selinux-icmp]]
+=== SELinux and ICMP permissions
+
+If you have SELinux enabled on your {page-component-title} server, there must be a policy configured to allow {page-component-title} to bind to the ICMP service.
+
+To verify that you have SELinux running and enabled, use the `sestatus` command.
```

Comment:
```suggestion
To verify that you have SELinux running and enabled, use the `sestatus` console command.
```

Reply:
I'd rather not specify all the types of commands, as we would have to update all the docs, and that's not a priority. Also, if you hover over the code sample in the docs, you see the type of command.